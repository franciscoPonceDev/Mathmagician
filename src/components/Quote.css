.quote {
  text-align: center;
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
}

.quote div {
  margin-top: 5rem;
  width: 60%;
}
