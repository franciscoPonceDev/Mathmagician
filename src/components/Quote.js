/* eslint-disable max-len */
import './Quote.css';

const Quote = () => (
  <section className="quote">
    <div>
      <h2>Mathemagics is not about numbers, equations,computations, or algorithms; Is about understanding. -William Paul Thurston</h2>
    </div>
  </section>
);

export default Quote;
