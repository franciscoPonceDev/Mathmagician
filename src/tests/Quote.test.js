import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Quote function', () => {
  it('quotes render correct', () => {
    const tree = renderer.create(<Quote />).toJSON();
  
    expect(tree).toMatchSnapshot();
  });
});
