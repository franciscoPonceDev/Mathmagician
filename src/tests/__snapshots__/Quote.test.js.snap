// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quotes render correct 1`] = `
<section
  className="quote"
>
  <div>
    <h2>
      Mathemagics is not about numbers, equations,computations, or algorithms; Is about understanding. -William Paul Thurston
    </h2>
  </div>
</section>
`;

exports[`quotes render correctly 1`] = `
<section
  className="quote"
>
  <div>
    <h2>
      Mathemagics is not about numbers, equations,computations, or algorithms; Is about understanding. -William Paul Thurston
    </h2>
  </div>
</section>
`;
